oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -46,106 +46,101 @@
-    private final AeadEncryptor encryptor;
-
-    /**
-     * Creates an attested gRPC channel.
-     * 
-     * `url` must contain protocol used for connection ("https://" or "http://").
-     */
-    public AttestationClient(String url) throws GeneralSecurityException, InterruptedException, MalformedURLException {
-        // Create gRPC channel.
-        URL parsedUrl = new URL(url);
-        if (parsedUrl.getProtocol().equals("https")) {
-            channel = ManagedChannelBuilder
-                .forAddress(parsedUrl.getHost(), parsedUrl.getPort())
-                .build();
-        } else {
-            channel = ManagedChannelBuilder
-                .forAddress(parsedUrl.getHost(), parsedUrl.getPort())
-                .usePlaintext()
-                .build();
-        }
-        RemoteAttestationStub stub = RemoteAttestationGrpc.newStub(channel);
-
-        // Create server response handler.
-        messageQueue = new ArrayBlockingQueue<>(1);
-        StreamObserver<AttestedInvokeResponse> responseObserver =
-                new StreamObserver<AttestedInvokeResponse>() {
-                    @Override
-                    public void onNext(AttestedInvokeResponse response) {
-                        try {
-                            messageQueue.put(response);
-                        } catch (Exception e) {
-                            if (e instanceof InterruptedException) {
-                                Thread.currentThread().interrupt();
-                            }
-                            logger.log(Level.WARNING, "Couldn't send server response to the message queue: " + e);
-                        }
-                    }
-
-                    @Override
-                    public void onError(Throwable t) {
-                        Status status = Status.fromThrowable(t);
-                        logger.log(Level.WARNING, "Couldn't receive response: " + status);
-                    }
-
-                    @Override
-                    public void onCompleted() {}
-                };
-        requestObserver = stub.attestedInvoke(responseObserver);
-
-        // Generate client private/public key pair.
-        AttestationEngine attestationEngine = new AttestationEngine(test_tee_measurement.getBytes());
-        AttestationIdentity identity = attestationEngine.getIdentity();
-
-        // Send client public key to the server.
-        AttestedInvokeRequest request =
-                AttestedInvokeRequest.newBuilder()
-                        .setClientIdentity(identity)
-                        .build();
-        requestObserver.onNext(request);
-        AttestedInvokeResponse response = messageQueue.take();
-
-        // Remotely attest peer.
-        AttestationIdentity peerIdentity = response.getServerIdentity();
-        encryptor = attestationEngine.createClientEncryptor(peerIdentity);
-    }
+  private static final Logger logger = Logger.getLogger(AttestationClient.class.getName());
```

Comment:
Is this really the correct formatting? No indentation? If so, we might be able to enable clang-formatting for java files. I experimented with it before, and I found the indentation weird, so I did not update the runner.

Reply:
According to https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation it should be 2 spaces at each level, rather than the 4 that is used for e.g. Rust.